x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_by_resource/severity_map.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { css } from '@emotion/css';
+import {
+  EuiColorPaletteDisplay,
+  EuiToolTip,
+  useEuiTheme,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { getSeverityStatusColor } from '../../../common/utils/get_vulnerability_colors';
+import { SeverityStatusBadge } from '../../../components/vulnerability_badges';
+
+interface Props {
+  total: number;
+  severityMap: SeverityMap;
+}
+
+interface SeverityMap {
+  critical: number;
+  high: number;
+  medium: number;
+  low: number;
+}
+
+const formatPercentage = (percentage: number) => {
+  if (percentage === 0) {
+    return '0 %';
+  }
+
+  if (percentage === 100) {
+    return '100 %';
+  }
+
+  return `${percentage.toFixed(1)} %`;
+};
```

Comment:
remove spaces between number and "%"
```suggestion
const formatPercentage = (percentage: number) => {
  if (percentage === 0) {
    return '0%';
  }

  if (percentage === 100) {
    return '100%';
  }

  return `${percentage.toFixed(1)}%`;
};
```

nit:
```typescript
if (percentage === 0 || percentage === 100) {
    return `${percentage.toFixed(0)}%`;
}
return `${percentage.toFixed(1)}%`;
```

Reply:
nice catch